.travis.yml

```diff
@@ -0,0 +1,44 @@
+language: cpp
+compiler: gcc
+sudo: required
+dist: trusty
+
+addons:
+ apt:
+  sources:
+   - ubuntu-toolchain-r-test
+  packages:
+   - python
+   - libsdl1.2-dev
+   - libsdl-image1.2-dev
+   - libgtkmm-2.4-dev
+   - g++-4.8
+   - libzmqpp-dev
```

Comment:
You actually only need python, no? Will be slightly faster not to install the rest.